core/src/test/java/org/apache/calcite/avatica/FrameTest.java

```diff
@@ -105,6 +106,84 @@ public void testMultipleRows() {
     serializeAndTestEquality(singleRow);
   }
 
+  protected void testValueRoundTrip(Object value) {
```

Comment:
Can you please update `testSingleRow` in this class to also use this helper method you've created?

Reply:
Sorry, @joshelser, I'm not sure what you mean.

`testSingleRow`, unlike `testValueRoundTrip`, uses a multi-value row and both call the existing `serializeAndTestEquality` method to do actual validation.

`testValueRoundTrip` is essentially a convenience method for deal with single-valued rows.

Reply:
Ah, sorry. I see. I was getting "one row" mixed up with "one value in that row". 👍